opentelemetry-api/src/opentelemetry/util/_importlib_metadata.py

```diff
@@ -12,27 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from sys import version_info
+from importlib_metadata import (  # type: ignore
```

Comment:
Can you leave a comment that this should be changed to `importlib.metadata` once 3.11 (or whichever last incompatible version) is dropped?

Reply:
Sure, let's hope it does not change again after 3.12 :laughing: 

<pre>
tigre@hilleman:~/github/ocelotl/opentelemetry-python$ python3.8
Python <b>3.8.16</b> (default, Feb 27 2023, 09:07:27) 
[GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>> type(entry_points())
<b>&lt;class 'dict'&gt;</b>
>>> 
tigre@hilleman:~/github/ocelotl/opentelemetry-python$ python3.9
Python <b>3.9.16</b> (main, Feb 24 2023, 17:23:48) 
[GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>> type(entry_points())
<b>&lt;class 'dict'&gt;</b>
>>> 
tigre@hilleman:~/github/ocelotl/opentelemetry-python$ python3.10
Python <b>3.10.10</b> (main, Feb 10 2023, 19:11:31) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>> type(entry_points())
<b>&lt;class 'importlib.metadata.SelectableGroups'&gt;</b>
>>> 
tigre@hilleman:~/github/ocelotl/opentelemetry-python$ python3.11
Python <b>3.11.2</b> (main, Feb 12 2023, 12:17:23) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>> type(entry_points())
<b>&lt;class 'importlib.metadata.SelectableGroups'&gt;</b>
>>> 
Python <b>3.12.0a5</b> (main, Mar 16 2023, 11:59:51) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.metadata import entry_points
>>> type(entry_points())
<b>&lt;class 'importlib.metadata.EntryPoints'&gt;</b>
</pre>

Reply:
Done